pymc3/distributions/multivariate.py

```diff
@@ -505,7 +505,13 @@ def __init__(self, n, p, *args, **kwargs):
     def _random(self, n, p, size=None):
         if size == p.shape:
             size = None
-        return np.random.multinomial(n, p, size=size)
+
+        if p.ndim > 1 and p.shape[0] > 1:
```

Comment:
so `p=np.asarray([[.25,.25,.25,.25]])` hits the second case, but that's sort of ok, since the returned sample will have the correct shape.